Notes

```diff
@@ -0,0 +1,5 @@
+Discord Token
```

Comment:
AYOOOOO. All the keys should be in the `.env` file. That way it stays private and it won't be mixed into the codebase.

Just delete this file.